packages/gatsby-image/src/index.js

```diff
@@ -416,7 +465,7 @@ class Image extends React.Component {
 
     if (fluid) {
       const imageVariants = fluid
-      const image = imageVariants[0]
+      const image = getCurrentSrcData({ fluid })
```

Comment:
I would just pass on `fluid` as an argument, an object seems overkill here.

Reply:
The reasoning behind using them as an object was just being able to throw in `props` if needed...